src/app/component/eco-news/filter-news/filter-news.component.css

```diff
@@ -1,5 +1,6 @@
 .wrapper {
     display: flex;
+  padding-top: 6px;
```

Comment:
Please, add 1 tab, because now you have bad alignment

Reply:
Fixed